package.json

```diff
@@ -11,6 +11,7 @@
     "travis-deps": "pip install -t lib -r requirements.travis.txt --upgrade",
     "dev-deps": "python -m pip install --no-deps -r requirements.dev.txt",
     "test": "python -m unittest discover -p '*_test.py'  -b",
+    "emulator" : "gcloud beta emulators datastore start --host-port=:15606 --no-store-on-disk",
     "coverage": "python -m pip install --no-deps -r requirements.dev.txt && python -m coverage erase && python -m coverage run -m unittest discover -p '*_test.py' -b && python -m coverage html",
```

Comment:
We can use this script to start the emulator. `--no-store-on-disk` will not store the entries on the disk.

Reply:
So, would the user need to run that command before running tests?  Can we put it into the `test` target?

Reply:
> So, would the user need to run that command before running tests?
Yes the developer will have to run the emulator before running any test.

> Can we put it into the test target?
On executing the npm run emulator, the terminal will get busy with the emulator process. To run the tests, we will have to use another terminal.

Reply:
OK, that sounds fine for now.   Later we can try running it in the background in the same npm target.